packages/services/f-feature-management/test/featureGetter.test.js

```diff
@@ -0,0 +1,118 @@
+import { getFeature, init, poll } from '../src/featureGetter';
+import fetchMock from "jest-fetch-mock";
```

Comment:
You might want to run the linter, it should suggest changing these to single quotes, among other things.

Reply:
I did, and it didn't